spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -8,6 +8,10 @@ def remove_index
     remove_file "public/index.html"
   end
 
+  def appease_webpacker
```

Comment:
Can you add a comment about why this needs to be here?

Reply:
Great idea. I've added in a comment. Thanks @jcoyne @jrochkind

Reply:
I thought sprockets looks for this to be present, not webpacker 🤔

Reply:
assuming (~perhaps~ likely incorrectly) that this is a fix for https://github.com/projectblacklight/blacklight/issues/2231#issuecomment-577372873